ceno_zkvm/src/structs.rs

```diff
@@ -205,7 +205,8 @@ impl<E: ExtensionField> ZKVMFixedTraces<E> {
 
 #[derive(Default)]
 pub struct ZKVMWitnesses<E: ExtensionField> {
-    pub witnesses: BTreeMap<String, RowMajorMatrix<E::BaseField>>,
+    witnesses_opcodes: BTreeMap<String, RowMajorMatrix<E::BaseField>>,
```

Comment:
Wouldn't we want the kinds instead of the opcodes?

Reply:
There can be other circuits. It is called "opcode" everywhere until we find a more precise name.

Reply:
So what's actually in there? The instructions or something else?